moveit2.repos

```diff
@@ -1,17 +1 @@
 repositories:
```

Comment:
I'm inclined to either not use this file for CI at all, but still list our dependencies so that they are included when building from source

Reply:
that works as well